packages/common-ui-web/src/icons/assets.ts

```diff
@@ -329,4 +329,8 @@ export const MonkIconAssetsMap: IconAssetsMap = {
     '<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill="#000" d="M15.499 14h-.79l-.28-.27a6.5 6.5 0 0 0 1.48-5.34c-.47-2.78-2.79-5-5.59-5.34-4.23-.52-7.78 3.04-7.27 7.27.34 2.8 2.56 5.12 5.34 5.59a6.5 6.5 0 0 0 5.34-1.48l.27.28v.79l4.26 4.25c.41.41 1.07.41 1.48 0l.01-.01c.41-.41.41-1.07 0-1.48L15.499 14Zm-6 0c-2.49 0-4.5-2.01-4.5-4.5S7.009 5 9.499 5s4.5 2.01 4.5 4.5-2.01 4.5-4.5 4.5Zm0-7c-.28 0-.5.22-.5.5V9h-1.5c-.28 0-.5.22-.5.5s.22.5.5.5h1.5v1.5c0 .28.22.5.5.5s.5-.22.5-.5V10h1.5c.28 0 .5-.22.5-.5s-.22-.5-.5-.5h-1.5V7.5c0-.28-.22-.5-.5-.5Z"/></svg>',
   'zoom-out':
     '<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill="#000" d="M15.5 14h-.79l-.28-.27a6.5 6.5 0 0 0 1.48-5.34c-.47-2.78-2.79-5-5.59-5.34a6.505 6.505 0 0 0-7.27 7.27c.34 2.8 2.56 5.12 5.34 5.59a6.5 6.5 0 0 0 5.34-1.48l.27.28v.79l4.26 4.25c.41.41 1.07.41 1.48 0l.01-.01c.41-.41.41-1.07 0-1.48L15.5 14Zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14Zm-2-5h4c.28 0 .5.22.5.5s-.22.5-.5.5h-4c-.28 0-.5-.22-.5-.5s.22-.5.5-.5Z"/></svg>',
+  'rotate-left':
+    '<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill="#000" d="M 7.8 13.5 c -0.1 -4.6 0.9 -7 2.5 -8.9 L 8.4 3.7 C 7.8 3.1 8.2 2 9.1 2 h 5.6 C 15.2 2 15.7 2.4 15.7 3 v 5.6 c 0 0.9 -1.1 1.3 -1.7 0.7 l -1.9 -1.9 c -1.2 1.4 -1.9 3.1 -1.9 5.1 C 10.2 15.6 11.9 18.4 14.7 19.7 c 0.6 0.3 0.8 0.9 0.6 1.5 C 15 21.9 14.2 22.2 13.6 21.9 c -3.6 -1.5 -3.9 -2.9 -5.3 -5.9 Z" /></svg>',
+  'rotate-right':
+    '<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill="#000" d="M 16.2 13.5 c 0.1 -4.6 -0.9 -7 -2.5 -8.9 l 1.9 -0.9 c 0.6 -0.6 0.2 -1.7 -0.7 -1.7 h -5.6 c -0.5 0 -1 0.4 -1 1 v 5.6 c 0 0.9 1.1 1.3 1.7 0.7 l 1.9 -1.9 c 1.2 1.4 1.9 3.1 1.9 5.1 c 0 2.1 -1.7 4.9 -4.5 6.2 c -0.6 0.3 -0.8 0.9 -0.6 1.5 c 0.2 0.5 0.9 0.9 1.5 0.6 c 3.6 -1.5 4.3 -2.8 5.3 -5.1 z" /></svg>',
```

Comment:
Did you remember to pass these SVG into SVGO before adding them to the code like that?

Reply:
applied now